app/views/admin/dashboard/index.html.erb

```diff
@@ -1 +1,27 @@
 <h1>Admin Dashboard</h1>
+
+<%= link_to "My Merchants", admin_merchants_path %>
+
+<%= link_to "My Invoices", admin_invoices_path %>
+
+
+
+<div id="top5">
+  Favorite Customers<% @customers.admin_favorite_customers.each do |id, count| %>
+  <div id="cust-<%= id %>">
+    <%= @customers.find(id).full_name %> with <%= count %> successful transactions.
```

Comment:
Would probably be better to have an instance variable you could call from the controller rather than doing an AR query in the view.